lib/src/helpers.dart

```diff
@@ -15,6 +17,11 @@ DeviceScreenType getDeviceType(
 
   if (kIsWeb) {
     deviceWidth = size.width;
+  } else {
+    //Check if the device is a desktop, should not be web
+    if (Platform.isMacOS || Platform.isWindows || Platform.isLinux) {
+      deviceWidth = size.width;
+    }
```

Comment:
Why not put all of them in one `if()` statement?

```suggestion
    if (kIsWeb || Platform.isMacOS || Platform.isWindows || Platform.isLinux) { 
      deviceWidth = size.width;
    }
```

Reply:
Good suggestion.